cache/global.go

```diff
@@ -1,37 +1,82 @@
 package cache
 
-import "github.com/newrelic/infra-integrations-sdk/log"
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/newrelic/infra-integrations-sdk/log"
+)
 
 // DefaultDebug default debug mode for the cache.
 const DefaultDebug = false
 
 // TODO delete when removing these globals
 var GlobalLog = log.NewStdErr(DefaultDebug)
 
-var instance, err = NewCache(GlobalLog)
-
 func SetupLogging(verbose bool) {
 	GlobalLog.SetDebug(verbose)
 }
 
+const integrationsDir = "nr-integrations"
```

Comment:
why this new behaviour?

Reply:
Just to have all the caches in a single directory, in case more than one integration coexist.